administrator/components/com_admin/sql/updates/postgresql/4.0.0-2021-05-07.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE "#__mail_templates" SET "subject" = 'COM_PRIVACY_EMAIL_DATA_EXPORT_COMPLETED_SUBJECT' WHERE "template_id" = 'com_privacy.userdataexport';
+UPDATE "#__mail_templates" SET "body" = 'COM_PRIVACY_EMAIL_DATA_EXPORT_COMPLETED_BODY' WHERE "template_id" = 'com_privacy.userdataexport';
```

Comment:
```suggestion
UPDATE "#__mail_templates"
   SET "subject" = 'COM_PRIVACY_EMAIL_DATA_EXPORT_COMPLETED_SUBJECT',
       "body" = 'COM_PRIVACY_EMAIL_DATA_EXPORT_COMPLETED_BODY'
 WHERE "template_id" = 'com_privacy.userdataexport';
```
See previous review comment.